consensus/hotstuff/verification/combined_signer_v2.go

```diff
@@ -0,0 +1,123 @@
+package verification
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/onflow/flow-go/consensus/hotstuff"
+	"github.com/onflow/flow-go/consensus/hotstuff/model"
+	"github.com/onflow/flow-go/consensus/hotstuff/signature"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module"
+)
+
+// CombinedSignerV2 creates votes for the main consensus. Per protocol specification
```

Comment:
I didn't get which module is `CombinedSigner` in the architecture proposal https://www.notion.so/dapperlabs/Architecture-Proposal-07c8a782e95c48cea67fa8caa1ea0e18.

Reply:
Since leader can sign blocks in a stateless manner, the combined signer is for consensus nodes to produce both signatures.

In V2, it makes sense to be called "Combined" signer, because we combine both staking sig and random beacon sig. And there is a "Single" signer, which is for collection cluster, who only sign staking sig always.

In V3, consensus leader only sign one sig, either staking and random beacon, yes, it's a bit weird to continue calling it "Combined" signer, since we are no longer combining any sig. But should be rename "Combined" signer to "Single" signer. Because "Single" signer is for the collection cluster. We might need to find a better name for V3...